### PR TITLE
Adding SELinux log forwarding for stage management clusters

### PR DESCRIPTION
**Pull request by npecka**
SREP has requested assistance in setting up SELinux logging in the management clusters for ROSA / HCP Infrastructure. This change targets the staging environment first.

Jira: https://issues.redhat.com/browse/MTSRE-1410

### CODE REVIEW

**Comment by fahlmant** [comment]
@npecka Would you mind adding the related Jira card for this in the PR description?

**Comment by eqrx** [comment]
Anything else that blocks this PR? Can I help?:)

**Review by sam-nguyen7 — COMMENTED** [review]
/lgtm

File: hack/olm-registry/olm-artifacts-template.yaml [hunk 1]
```diff
@@ -500,6 +500,10 @@ objects:
               index: openshift_managed_audit_stage
               whiteList: \.log$
               sourceType: _json
+            - path: /host/var/log/audit/audit.log
+              index: rh_osd_selinux
```
Comment (karthikperu7) [inline hack/olm-registry/olm-artifacts-template.yaml:504]: I think we should wait for https://issues.redhat.com/browse/OSD-17654 to be complete. The index name would most probably be different to what's here.

**Pull request merged**